### PR TITLE
Add `containerd` image

### PR DESCRIPTION
**Pull request by afdesk**
Added a test containerd image.

### CODE REVIEW

File: containerd/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM ghcr.io/westonsteimel/containerd:1.6.1 as source
+
+FROM alpine
+COPY --from=source /usr/local/bin/ /usr/local/bin/
+RUN apk add bash
+RUN ls /usr/local/bin
```
Comment (knqyf263) [inline containerd/Dockerfile:6]: Do we need them?

File: containerd/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM ghcr.io/westonsteimel/containerd:1.6.1 as source
+
+FROM alpine
+COPY --from=source /usr/local/bin/ /usr/local/bin/
```
Comment (knqyf263) [inline containerd/Dockerfile:4]: If we copy only containerd binary, doesn't it work?

File: containerd/Dockerfile (hunk 1 above)
Comment (afdesk) [reply containerd/Dockerfile]: removed

File: containerd/Dockerfile (hunk 2 above)
Comment (afdesk) [reply containerd/Dockerfile]: this folder contains next files:
```sh
$ ls /usr/local/bin
containerd
containerd-shim
containerd-shim-runc-v1
containerd-shim-runc-v2
containerd-stress
```

so it seems yes, we need only `containerd`, isn't it?

File: containerd/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+FROM ghcr.io/westonsteimel/containerd:1.6.1 as source
+
+FROM alpine
+COPY --from=source /usr/local/bin/containerd /usr/local/bin/
+RUN apk add bash
```
Comment (knqyf263) [inline containerd/Dockerfile:5]: I guess bash is also not needed.

Comment (afdesk) [reply containerd/Dockerfile]: you're right. it doesn't need. removed

**Pull request merged**